index.css

```diff
@@ -0,0 +1,169 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
Good use of grid to position elements on the page. There are few feedbacks about the page: 
1. Bird image is positioned similarly but only on blue div. On actual site, it is positioned in center of the page. 
2. You could try fonts that are closer looking to the actual website. 
3. Texts in blue section could use icons and perhaps bigger font. 
4. Login inputs and buttons on white section could use centering and heights/widths similar to the Twitter page. 
5. No favicon for the page (mini icon on browser tab)
6. Page could respond better to re-sizing of browser window. (Flex is useful for this feature)
Overall this is pretty similar to Twitter page with few improvements in mind. Otherwise good work and looking forward to see your next assignment with Pokemons.